sys-devel/gettext/gettext-0.22.5-r1.ebuild

```diff
@@ -7,7 +7,7 @@ EAPI=8
 
 VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/gettext.asc
 inherit java-pkg-opt-2 libtool multilib-minimal verify-sig toolchain-funcs
-inherit flag-o-matic
+inherit autotools flag-o-matic
```

Comment:
This may cause issues for e.g. Prefix bootstrap as it'll depend on automake and such too. We could do `AUTOTOOLS_AUTO_DEPEND=no` and depend on just autoconf manually but it's better to add a patch which includes the generated changes to configure too for such core packages.

Reply:
Maybe it's better to just leave it for older gettext and we try to cleanup old. I think just one Python package was blocking it.